tests/test_inband_intf_mgmt_vrf.py

```diff
@@ -0,0 +1,160 @@
+        assert len(current_entries) == 1
+
+    def create_inband_intf(self, interface):
+        cfg_tbl = cfg_key = cfg_fvs = None
+        if interface.startswith('PortChannel'):
+            tbl_name = 'PORTCHANNEL_INTERFACE'
+            cfg_tbl = 'PORTCHANNEL'
+            cfg_key = interface
+            cfg_fvs = swsscommon.FieldValuePairs([("admin_status", "up"),
+                                              ("mtu", "9100")])
+        elif interface.startswith('Vlan'):
+            tbl_name = 'VLAN_INTERFACE'
+            cfg_tbl = 'VLAN'
+            vlan_id = interface[len('Vlan'):]
+            cfg_key = 'Vlan' + vlan_id
+            cfg_fvs = swsscommon.FieldValuePairs([("vlanid", vlan_id)])
+        elif interface.startswith('Loopback'):
+            tbl_name = 'LOOPBACK_INTERFACE'
+        else:
+            tbl_name = 'INTERFACE'
+        if cfg_tbl is not None:
+            tbl = swsscommon.Table(self.cfg_db, cfg_tbl)
+            tbl.set(cfg_key, cfg_fvs)
+            time.sleep(1)
+        fvs = swsscommon.FieldValuePairs([('vrf_name', MGMT_VRF_NAME)])
+        tbl = swsscommon.Table(self.cfg_db, tbl_name)
+        tbl.set(interface, fvs)
+        time.sleep(1)
+
+    def remove_inband_intf(self, interface):
+        cfg_tbl = cfg_key = None
+        if interface.startswith('PortChannel'):
+            tbl_name = 'PORTCHANNEL_INTERFACE'
+            cfg_tbl = 'PORTCHANNEL'
+            cfg_key = interface
+        elif interface.startswith('Vlan'):
+            tbl_name = 'VLAN_INTERFACE'
+            cfg_tbl = 'VLAN'
+            vlan_id = interface[len('Vlan'):]
+            cfg_key = 'Vlan' + vlan_id
+        elif interface.startswith('Loopback'):
+            tbl_name = 'LOOPBACK_INTERFACE'
+        else:
+            tbl_name = 'INTERFACE'
+        tbl = swsscommon.Table(self.cfg_db, tbl_name)
+        tbl._del(interface)
+        time.sleep(1)
+        if cfg_tbl is not None:
+            tbl = swsscommon.Table(self.cfg_db, cfg_tbl)
+            tbl._del(cfg_key)
+            time.sleep(1)
+
+    def test_MgmtVrf(self, dvs, testlog):
+        self.setup_db(dvs)
+
+        vrf_oid = self.add_mgmt_vrf()
+        self.del_mgmt_vrf()
+
+    @pytest.mark.parametrize('intf_name', ['Ethernet4', 'Vlan100', 'PortChannel5', 'Loopback1'])
+    def test_InbandIntf(self, intf_name, dvs, testlog):
+        self.setup_db(dvs)
+
+        vrf_oid = self.add_mgmt_vrf()
+        self.create_inband_intf(intf_name)
+
+        # check application database
+        tbl = swsscommon.Table(self.appl_db, 'INTF_TABLE')
+        intf_keys = tbl.getKeys()
+        status, fvs = tbl.get(intf_name)
+        assert status == True
+        for fv in fvs:
+            if fv[0] == 'vrf_name':
+                assert fv[1] == MGMT_VRF_NAME
+
+        if not intf_name.startswith('Loopback'):
+            # check ASIC router interface database
+            tbl = swsscommon.Table(self.asic_db, 'ASIC_STATE:SAI_OBJECT_TYPE_ROUTER_INTERFACE')
```

Comment:
for checking ASIC db, could you please use https://github.com/Azure/sonic-swss/blob/master/tests/test_mux.py#L90
Using sleep may be flaky